docs/en/latest/admin-api.md

```diff
@@ -103,6 +103,33 @@ deployment:
 
 This will find the environment variable `ADMIN_KEY` first, and if it does not exist, it will use `edd1c9f034335f136f87ad84b625c8f1` as the default value.
 
+### Force Delete
+
+By default, the Admin API checks for references between resources and will refuse to delete resources in use.
+
+You can make a force deletion by adding the request header `X-Force-Delete: true` to the delete request, for example:
```

Comment:
Is it better to put this in `uri_args` ?

Reply:
I have no preference for this design. wdyt @kingluo @leslie-tsang

Reply:
If you put the configuration into the uri_args, then you don't need to add this following code
```
        if method == "delete" then
            if not req_body then
                req_body = {}
            end
            req_body._force_delete = ngx.req.get_headers()["X-Force-Delete"]
        end
```

Reply:
I prefer placing the parameter in `uri_args` to keep this interface as simple as possible. :)